README.md

```diff
@@ -1,2 +1,21 @@
+---
+output: html_document
+---
 # hw01
 Repository for first CFSS homework assignment
+
+# Introduction
+I am a graduate student in the _MAPSS_ program with an economics concentration. 
+The list of classes that I am taking this quarter include:
+
+1. Firm Dynamics and Innovation
+2. Analysis of Micro Data
+3. This Class
+
+You can learn more about the MAPSS program through its website:
+https://mapss.uchicago.edu/
+
+# Report
+I created a new project from version control and pulled from github using git pull command. Then I modified the markdown file in Rstudio. 
+
+The hard problem is to find out the R Markdown code seems to require blank line before the list in order for list syntax to work properly. Trying to understand how commit and push work is also the harder part. The other parts are relatively easy. 
```

Comment:
I also had issues with creating lists and it seems like others did too.  Just like you I found that using a blank line in between helps.
Based on my time reviewing other's work it seems that commits will save versions of your work to GitHub for review if needed.